community/knative/charts/eventing/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "0.2.1"
+description: A Helm chart for Kubernetes
+name: eventing
+version: 0.1.0
+icon: https://raw.githubusercontent.com/knative/docs/master/images/logo/rgb/knative-logo-rgb.jpg
+kubeVersion: ">=1.10.3"
+tillerVersion: ">=2.10.0"
```

Comment:
Needs to include at least 2.9.1 for support on 3.1.2

Reply:
same as https://github.com/IBM/charts/pull/184#discussion_r255170249